scripts/build.py

```diff
@@ -113,7 +113,8 @@
     'third_party/generated/webfonts/*',
     '*.bundle.js',
     '*.bundle.js.map',
-    'webpack_bundles/*'
+    'webpack_bundles/*',
+    'third_party/generated/proto/.gitkeep*'
```

Comment:
Can you please add an explanation here (in GitHub comment) for these changes i.e, why it's added here?

Reply:
The gitkeep file is there to keep the `third_party/generated/proto/` path alive in GitHub repo because it is populated with auto-generated protobuf files. Check the readme file that I added in `extensions/classifier` directory. Basically, this file should be ignored by build process.

Reply:
This gitkeep file should not be included in the build process but I added the exception at wrong place. I removed this and added `.gitkeep` in the extensions list rather than this path.